.gitignore

```diff
@@ -17,6 +17,7 @@ build/
 internal/venice-test-common/tmp/
 internal/venice-test-common/dump.complete.hprof
 internal/venice-test-common/src/jmh/generated
+internal/venice-test-common/file:/
```

Comment:
minor question - why do we need to add this?

Reply:
When running the E2E integration tests locally, I saw this folder got created every time and that is the reason I would like to exclude from here.